pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/export/json/structured/QuestionRecord.java

```diff
@@ -34,7 +34,7 @@ public JsonElement serialize(QuestionRecord record, Type typeOfSrc, JsonSerializ
             if (record instanceof SimpleQuestionRecord) {
                 questionAnswer = ((SimpleQuestionRecord) record).getAnswer();
             } else if (record instanceof DateQuestionRecord) {
-                questionAnswer = ((DateQuestionRecord) record).getDate();
+                questionAnswer = ((DateQuestionRecord) record).getAnswer();
```

Comment:
I just went through to all kind of questions we have on `DSS` side and found that `MatrixQuestionRecord` values go under `matrixSelected`, maybe it is good time here as well to add `answer` property to `MatrixQuestionRecord` class as well? @SimoneMaiwald